### PR TITLE
Migrated TC test_snap_rebalance

### PR DESCRIPTION
**Pull request by nik-redhat**
Migrated TC [test_snap_rebalance](https://github.com/gluster/glusto-tests/blob/master/tests/functional/snapshot/test_snap_rebalance.py)

Updates: #292 
Signed-off-by: nik-redhat <nladha@redhat.com>

### CODE REVIEW

**Comment by nik-redhat** [comment]
```
Results for snapshot
Test nature : disruptive
TestName : test_snap_rebalance
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:01:23       |      NA     |
|   dist-rep  |     PASS    |        00:01:58       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   2   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 03 minutes 33 seconds
✔ Tearing down successful.
```

File: tests/functional/snapshot/test_snap_rebalance.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                            self.client_list[0])
+
+        # Validate clone volume mounted or not
+        if not redant.is_mounted(self.clone, self.mount1, self.client_list[0],
+                                 self.server_list[0]):
+            raise Exception("Cloned Volume not mounted on mount point: "
+                            f"{self.mount1}")
+
+        # write files to mountpoint
+        all_mounts_procs = []
```
Comment (Sheetalpamecha) [inline tests/functional/snapshot/test_snap_rebalance.py:99]: nit: no need of `all_mounts_procs = []` for single process, can be done with `proc` only

**Review by srijan-sivakumar — DISMISSED** [review]

File: tests/functional/snapshot/test_snap_rebalance.py (hunk 1 above)
Comment (nik-redhat) [reply tests/functional/snapshot/test_snap_rebalance.py]: Done

**Comment by nik-redhat** [comment]
```
Results for snapshot
Test nature : disruptive
TestName : test_snap_rebalance
+-------------+-------------+-----------------------+-------------+
| Volume Type | Test Result | Time Taken (hh:mm:ss) | Skip Reason |
+-------------+-------------+-----------------------+-------------+
|     dist    |     PASS    |        00:01:23       |      NA     |
|   dist-rep  |     PASS    |        00:01:43       |      NA     |
+-------------+-------------+-----------------------+-------------+
+--------------------------------------+-------+
|               Category               | Count |
+--------------------------------------+-------+
|           Disruptive Tests           |   1   |
|         Non Disruptive Tests         |   0   |
|       Disruptive Tests Skipped       |   0   |
|     Non Disruptive Tests Skipped     |   0   |
|              Tests Ran               |   2   |
|            Tests Skipped             |   0   |
|             Total Tests              |   1   |
|   Disruptive Tests Pass Percentage   | 100.0 |
| Non Disruptive Tests Pass Percentage |   0   |
|        Total Pass Percentage         | 100.0 |
+--------------------------------------+-------+
Total Time Taken : 00 hours 03 minutes 18 seconds
✔ Tearing down successful.
```

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**